### PR TITLE
ci(Build And Deploy): Uses danger to deploy

### PR DESCRIPTION
**Pull request by brad-decker**
### Yay for deployments
1. Deploys PR apps and aliases them to a known url for ease of navigating.
2. Deploys Master branch to https://styled-material-components.com
3. Lists modified components in PR notes

### CODE REVIEW

File: dangerfile.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+let markdown = '';
+
+if (modifiedComponents.length > 0) {
+  markdown += `
+    ### Modified Components
+    * ${modifiedComponents.join('\n * ')}
+
+  `;
```
Comment (martahj) [inline dangerfile.js]: [Q] Thoughts on doing something similar with mixins, theme, etc?

**Comment by ashwoodall** [comment]
Should we add a note about this in the README?

File: dangerfile.js (hunk 1 above)
Comment (brad-decker) [reply dangerfile.js]: I like it.

Comment (brad-decker) [reply dangerfile.js]: Do you mind if i tackle in a different PR? It could get pretty complex and i don't wanna block it here

**Comment by brad-decker** [comment]
@ashwoodall updated contributing.md to remove the manual deploy steps :D

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,12 +29,9 @@ Additionally, there are also many ways you can directly contribute, such as:
...
 0. Add the components to pages in /pages (Pages are mapped to routes for our documentation)
-0. Add a link to your new component page in index.js 
-0. Install [now](https://zeit.co/now#whats-now) if you do not already have it installed.
-0. Run `yarn now-build` to get a shareable link to the updated docs copied to your
-clipboard.
-0. Open a PR, making sure to include the link to the updated docs so others can
-see your wonderful changes.
+0. Add a link to your new component page in index.js
+0. Open a PR
+0. Our danger script will build your application and post a link as a comment to view your work!
```
Comment (AriLFrankel) [inline CONTRIBUTING.md]: 🎉

File: dangerfile.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  R.match(PR_URL),
+  R.view(STDOUT),
+  targetApp => exec(`yarn now -n ${targetApp} -d docs/ -t ${process.env.NOW_TOKEN}`),
+);
+
+const branch = process.env.CIRCLE_BRANCH || danger.github.pr.head.label;
+const appName = branch === 'master' ? NAME : `${NAME}-pr-${danger.github.pr.number || 0}`;
+
+// Get the modified components in the PR
+const getModifiedComponents = R.compose(
```
Comment (AriLFrankel) [inline dangerfile.js]: 👍

**Pull request merged**